include/zephyr/bluetooth/services/ots.h

```diff
@@ -867,6 +873,18 @@ struct bt_ots_client_cb {
 	void (*obj_metadata_read)(struct bt_ots_client *ots_inst,
 				  struct bt_conn *conn, int err,
 				  uint8_t metadata_read);
+	/** @brief Callback function for the data of the write
+	 * object.
+	 *
+	 *  Called when the data of the selected object are read using
```

Comment:
is? I know data can be plural, but you use "Length of the written data" later

Reply:
Revised.